recipes/oso/bld.bat

```diff
@@ -2,10 +2,10 @@
 
 cargo build -p polar-c-api
 
-cd $SRC_DIR\languages\python\oso
+cd %SRC_DIR%\languages\python\oso
```

Comment:
stricly speaking, the slashes would be `/` as well... though it generally _will_ work this way